R/check_ae_aeacn_ds_disctx_covid.R

```diff
@@ -65,33 +60,36 @@
 #' AE2$AEACN <- "MULTIPLE" 
 #' 
 #' # expect fail
-#' check_ae_aeacn_ds_disctx_covid(AE2, DS, covid_df)
+#' check_ae_aeacn_ds_disctx_covid(AE2, DS)
 #' 
 #' DS[1, "DSDECOD"] <- 'ADVERSE EVENT'
 #' # this passes, one form with DSDECOD = "ADVERSE EVENT"
 #' ## NOTE: This may be a false negative if study-specific data collection 
 #' ##       requires >1 record with DSDECOD = "ADVERSE EVENT" and not just one record
-#' check_ae_aeacn_ds_disctx_covid(AE2, DS, covid_df)
+#' check_ae_aeacn_ds_disctx_covid(AE2, DS)
 #'
 #' # expect pass
-#' check_ae_aeacn_ds_disctx_covid(AE, DS, covid_df)
+#' check_ae_aeacn_ds_disctx_covid(AE, DS)
 #' 
 #'
 #' # non-required variable is missing
 #' DS$DSSEQ <- NULL
-#' check_ae_aeacn_ds_disctx_covid(AE, DS, covid_df)
+#' check_ae_aeacn_ds_disctx_covid(AE, DS)
 #' 
 #' # required variable is missing 
 #' DS$DSSPID <- NULL
-#' check_ae_aeacn_ds_disctx_covid(AE, DS, covid_df)
+#' check_ae_aeacn_ds_disctx_covid(AE, DS)
 #' 
 #' 
 
-check_ae_aeacn_ds_disctx_covid <- function(AE,DS,covid_df = NULL) {
+check_ae_aeacn_ds_disctx_covid <- function(AE,DS,covid_terms=c("COVID-19", "CORONAVIRUS POSITIVE")) {
 
-    if (is.null(covid_df)){
+    if(is.null(covid_terms)|
+       (!is.null(covid_terms) & !is.vector(covid_terms))|
```

Comment:
@bodachs any feelings on using `is.vector` here?  I wanted to do some checks against supplying unexpected objects.  One thing that gives me pause is that `is.vector` is only `TRUE` when the object doesn't have any attributes.  When pulling a column from a dataframe it can happen where the resulting vector has a label attribute.  maybe just use `is.character` here instead.  Lets chat tomorrow.

Reply:
@bodachs FYI I think this code prevents a breaking change